src/views/homepage/components/certifications-section/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { CertificationsSectionProps } from './types'
+import {
+	CollectionCardPropsWithId,
+	CollectionCardWithAuthElements,
+} from 'components/collection-card'
+import CardsGridList from 'components/cards-grid-list'
+import { CertificationsTextAndImage } from './components/certifications-text-and-image'
+import s from './certifications-section.module.css'
+
+/**
+ * Render a section with a link to certifications content,
+ * and render some collection cards related to certifications.
+ */
+function CertificationsSection({
```

Comment:
The section component uses some underlying components. Styles for this section are pretty straightforward - they offset the `.collectionCards` section to overlap the text-image section.